src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,26 @@
-export const PostInfo = () => <>Put the post here</>;
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+export const PostInfo = ({ post: { title, body, user, comments } }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={user} />
+      </p>
+    </div>
+
+    <p className="PostInfo__body">{body}</p>
+
+    <hr />
+
+    {comments.length ? (
```

Comment:
in this case you can see 0 on FE while comments loading
we should directly make it boolean
if you don't want use !! you can use Boolean class
```suggestion
    {Boolean(comments.length) ? (
```